.github/workflows/performance-testing.yml

```
@@ -0,0 +1,419 @@
+        uses: actions/upload-artifact@v4
+        if: always()
+        with:
+          name: performance-summary-${{ github.run_number }}
+          path: performance-summary.json
+          retention-days: 90
+
+  performance-alerts:
+    name: Performance Alerts
+    runs-on: ubuntu-latest
+    needs: [performance-summary]
+    if: always() && (needs.performance-summary.result == 'failure' || contains(needs.performance-summary.outputs.status, 'FAIL'))
```

actionlint (1.7.7)

property "status" is not defined in object type {}

(expression)